### ISSUE TITLE
Add updating `git submodule` on first run to instructions

### ISSUE THREAD

**Issue by sudhanshu2** [hpcgarage/spatter#66]
Building without initializing `git submodule` leads to the following error
```
The source directory
    /spatter/external/argtable3
does not contain a CMakeLists.txt file.
```

This error is fixed by running the following commands in the project directory
```
git submodule init
git submodule update
```

It would be nice if running these commands were a part of the readme.

**Comment by jyoung3131** [comment]
Hi, Sudanshu,

Can you please add this info to the README or the testing wiki page and do a PR for it?

**Comment by jyoung3131** [comment]
Addressed with PR #68.
